### ISSUE TITLE
MU4: Opening MU3 scores including parts with all formatting from MU3 

### ISSUE THREAD

**Issue by Rabadman** [musescore/MuseScore#23073]
### Your idea

Importing scores including existing parts from MuseScore 3 and older into Musescore 4
Many existing parts from Musescore 3 and older currently have to be recreated in Musescore 4.

### Problem to be solved

Easier to switch from Musescore 3 to Musescore 4

**Comment by Jojo-Schmitz** [comment]
Any sample score that shows this?

**Comment by Jojo-Schmitz** [comment]
If there is, this wouldn't be a feature request, but a bug report

**Comment by Rabadman** [comment]
Hello Jojo,

glad to hear from you :-)

I thought that the import of parts from MU3 files is deliberately not yet included in MU 4. Hence my request as a future feature.
Enclosed is the shortened score from MU3.
The screenshot opened once with MU3 and once with MU4. In MU4 no parts are displayed.

[Nie mehr ohne di sein.zip](https://github.com/user-attachments/files/15544355/Nie.mehr.ohne.di.sein.zip)
![Parts in MU3](https://github.com/musescore/MuseScore/assets/148609589/dc32d506-baf1-477a-8bf6-64a6334729a2)
![Without parts in MU4](https://github.com/musescore/MuseScore/assets/148609589/01795d10-1c4d-4c2b-bfea-b81dcccdd239)

**Comment by Jojo-Schmitz** [comment]
Yes, they are not displayed by default, but still available under the "Auszüge" button

**Comment by Rabadman** [comment]
That's right, the part excerpts can of course be displayed again in MU4.
But what an effort that would be with the many existing scores from MU3...

Now that MU3 is no longer being developed further, it would be desirable if the parts from MU3 files were also displayed in the tabs in MU4. For this and other reasons, I personally continue to work with your version 3.7

**Comment by Jojo-Schmitz** [comment]
So the feature request is: on importing Mu3 and Mu2 scores (Mu1 didn't have parts), open the parts that were open in those previous versions (but only those parts)

**Comment by Jojo-Schmitz** [comment]
Better clarify the issue title.

**Comment by cbjeukendrup** [comment]
Does https://github.com/musescore/MuseScore/pull/30685 solve this? (See [How to download test builds from pull requests](https://github.com/musescore/MuseScore/wiki/Downloading-and-running-test-builds#downloading-builds-from-pull-requests))

**Comment by Jojo-Schmitz** [comment]
I like it!

**Comment by Jojo-Schmitz** [comment]
Possible addition: In the Export dialog have an extra "Select all __open__ parts" between "Select all" and "Clear selection".
And/or mark those bold (like done for the "Main score"), so you find them easier.
But that probably is a separate issue
